SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/MobileCenter.cs

```diff
@@ -0,0 +1,402 @@
+﻿using Microsoft.Azure.Mobile.Channel;
+using Microsoft.Azure.Mobile.Ingestion.Models;
+using Microsoft.Azure.Mobile.Utils;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.Azure.Mobile
+{
+    /// <summary>
+    /// SDK core used to initialize, start and control specific service.
+    /// </summary>
+    public partial class MobileCenter
+    {
+        // Internals for testing
+        internal const string EnabledKey = "MobileCenterEnabled";
```

Comment:
I would create a Mobile Center prefix and then use it on all these keys for instance prefix + "CenterEnabled". I am quite sure there will be a lot more persistence keys like these.